platforms-serverless/serverless-framework-lambda/prisma/schema.prisma

```diff
@@ -1,7 +1,7 @@
 generator client {
+  output = "client"
```

Comment:
Isn't that buggy right now?

Reply:
Serverless framework does not understand pnpm, I could switch the project back to npm (not yarn please).

Reply:
(can this be marked as resolved?)

Reply:
Ok, lambda permission didn't allow me to do that easily, so testing via npm only now